Godeps/_workspace/src/k8s.io/kubernetes/pkg/apiserver/api_installer.go

```diff
@@ -126,13 +126,21 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	if err != nil {
 		return nil, err
 	}
-	versionedPtr, err := a.group.Creater.New(a.group.Version, kind)
+	serializationGroupVersionOfKind := a.group.Version
```

Comment:
I'm not really in favor of this change.  I want the rest storage to accept different input.